index.ts

```diff
@@ -99,14 +100,16 @@ const loadArticles = async (
       'content-type': 'application/json',
       authorization: apiKey,
     },
-    body: `{"query":"\\n    query Search($after: String, $first: Int, $query: String) {\\n      search(first: $first, after: $after, query: $query) {\\n        ... on SearchSuccess {\\n          edges {\\n            node {\\n              title\\n              slug\\n              url\\n              author\\n              description\\n            }\\n          }\\n          pageInfo {\\n            hasNextPage\\n          }\\n        }\\n        ... on SearchError {\\n          errorCodes\\n        }\\n      }\\n    }\\n  ","variables":{"after":"${after}","first":${first}, "query":"${
-      savedAfter ? 'saved:' + savedAfter : ''
-    } sort:saved-asc"}}`,
+    body: `{"query":"\\n    query Search($after: String, $first: Int, $query: String) {\\n      search(first: $first, after: $after, query: $query) {\\n        ... on SearchSuccess {\\n          edges {\\n            node {\\n              title\\n              slug\\n              url\\n              author\\n              updatedAt\\n              description\\n            }\\n          }\\n          pageInfo {\\n            hasNextPage\\n          }\\n        }\\n        ... on SearchError {\\n          errorCodes\\n        }\\n      }\\n    }\\n  ","variables":{"after":"${after}","first":${first}, "query":"sort:saved-asc"}}`,
     method: 'POST',
   })
 
   const jsonRes = (await res.json()) as SearchResponse
-  const articles = jsonRes.data.search.edges.map((e) => e.node)
+  const articles = jsonRes.data.search.edges
```

Comment:
This could be done with a change of query from "saved:<some date>" to "updated:<some date>"

Reply:
Ah that's what i was trying to find!!!
You might want to update the docs at https://omnivore.app/help/search 
I was trying to find the docs for the API to find a query for that!!

Reply:
Feel free if you want to work on this branch to make small adjustment or I can if you want?
I am happy with whatever
just let me know

Reply:
Ah thanks. Yeah I think we are going to move all these docs into Logseq so its easier to keep them up to date.